plugins/rbac/src/hooks/useRoles.ts

```diff
@@ -138,6 +186,7 @@ export const useRoles = (
         (typeof policies === 'object'
           ? (policies as any as Response)?.statusText
           : '')) as string,
+      roleConditionError: fetchRoleConditionsError,
```

Comment:
@ciiay can we pass this in the error object here https://github.com/janus-idp/backstage-plugins/blob/b83a315113ef69c5a85f0a100d730a99a4590b48/plugins/rbac/src/hooks/useRoles.ts#L172 instead of creating a new param for it.